src/Analysis/Ast/Impl/Analyzer/Symbols/ClassEvaluator.cs

```diff
@@ -50,22 +52,13 @@ internal sealed class ClassEvaluator : MemberEvaluator {
                 EvaluateInnerClasses(_classDef);
 
                 _class = classInfo;
-                // Set bases to the class.
-                var bases = new List<IPythonType>();
-                foreach (var a in _classDef.Bases.Where(a => string.IsNullOrEmpty(a.Name))) {
-                    // We cheat slightly and treat base classes as annotations.
-                    var b = Eval.GetTypeFromAnnotation(a.Expression);
-                    if (b != null) {
-                        var t = b.GetPythonType();
-                        bases.Add(t);
-                        t.AddReference(Eval.GetLocationOfName(a.Expression));
-                    }
-                }
-                _class.SetBases(bases);
 
+                List<IPythonType> bases = new List<IPythonType>();
```

Comment:
This seems weird; why doesn't `ProcessBases` just return an `IEnumerable`, instead of passing in an empty `List` to be modified?

Reply:
I was trying to follow the ```Process...``` style, where they were void methods - but you're right I thought it was weird too.